DDDEastAnglia/DataAccess/ConferenceLoader.cs

```diff
@@ -12,7 +12,7 @@ public interface IConferenceLoader
 
     public class ConferenceLoader : IConferenceLoader
     {
-        private const string DefaultEventName = "DDDEA2014";
+        private const string DefaultEventName = "DDDEA2015";
```

Comment:
Is this going to screw up the production site at all? E.g., it's currently closed, will this change open it again?